Library/Formula/xcconf.rb

```diff
@@ -0,0 +1,13 @@
+class Xcconf < Formula
+  homepage "https://github.com/AlexDenisov/xcconf"
+  url "https://github.com/AlexDenisov/xcconf/archive/0.1.1.tar.gz"
+  sha1 "d3768af9ab739c49ac107393ac19047a962e703a"
+
+  def install
+    system "make", "build"
+    bin.install "./build/Release/xcconf"
```

Comment:
Your Makefile should accept a `PREFIX` variable and then we can do `make PREFIX=#{prefix} install` and you can copy the file to `#{PREFIX}/bin` and create the `bin` directory. It's fairly standard for open-source projects.